packages/common/src/store/pages/profile/types.ts

```diff
@@ -58,11 +58,11 @@ export type ProfilePageState = {
 }
 
 export enum ProfilePageTabs {
-  TRACKS = 'TRACKS',
-  ALBUMS = 'ALBUMS',
-  PLAYLISTS = 'PLAYLISTS',
-  REPOSTS = 'REPOSTS',
-  COLLECTIBLES = 'COLLECTIBLES'
+  TRACKS = 'tracks',
```

Comment:
is there a reason for the case changes here (and the other files in general)

Reply:
Yeah we were using these values to lowercase because these values are used for the labels of the various tabs.